hw/formal/README.md

```diff
@@ -211,7 +211,7 @@ There are also powerful repetition operators, see
 
 When design has a set of modules or signals that share same properties,
 symbolic variables can be used to reduce duplicated assertions.
-For example, in the [rv_plic design](../ip/rv_plic/doc/rv_plic.md), the array of
+For example, in the [rv_plic design](../ip/rv_plic/doc/_index.md), the array of
```

Comment:
Good catch, but I think this should be a relref to the actual spec:

[rv_plic_design]({{<relref "hw/ip/rv_plic/doc">}})

Reply:
I did not want to change that as I wasn't clear if the relref should be done in README.md files as this would invalidate the github links.
That being said, there is a mixture of those links, so maybe we want to stick with one format.
Ideally that would be links which are valid in github and docs.opentitan

Reply:
makes sense for a README